catalog/app/constants/routes.ts

```diff
@@ -262,11 +262,23 @@ export const admin: Route = {
 
 export const adminUsers = admin
 
-export type AdminBucketsArgs = [bucket: string]
+export type AdminBucketLegacyArgs = [bucket: string]
 
-export const adminBuckets: Route<AdminBucketsArgs> = {
+export const adminBuckets: Route<AdminBucketLegacyArgs> = {
   path: '/admin/buckets',
-  url: (bucket) => `/admin/buckets${mkSearch({ bucket })}`,
+  url: (legacyBucketParam) => `/admin/buckets${mkSearch({ bucket: legacyBucketParam })}`,
+}
+
+export type AdminBucketEditArgs = [bucket: string]
+
+export const adminBucketEdit: Route<AdminBucketEditArgs> = {
+  path: '/admin/buckets/edit/:bucketName',
```

Comment:
this url suggests that there should be something under `/admin/buckets/edit/` route, but there is nothing,
which is confusing

Reply:
Do you have another idea for a route?

Also, just out of my head several examples of such routes:

* https://github.com/quiltdata/quilt/pull/
* https://nightly.quilttest.com/b/allencell/packages/aics/
* https://nightly.quilttest.com/b/

I mean, I don't think this is ideal, but also I don't think it is expected that every "slash" has something

Reply:
> i was thinking about `/buckets/$name` vs `/buckets?add`

Ok, let's do this